managing-os/windows/windows-vm.md

```diff
@@ -8,6 +8,61 @@ permalink: /doc/windows-vm/
 Installing a Windows VM
 =======================
 
+Simple Windows 10 install (Qubes 4.0)
+-------------------------------------
+
+If you just want something simple and you can live without some features.
+
+Works:
+- display (1440x900 or 1280x1024 are a nice fit onto FHD hw display)
+- keyboard (incl. correct mapping), pointing device
+- network (emulated Realtek NIC)
+
+Does not work:
+- copy & paste (the qubes way)
+- copying files into / out of the VM (the qubes way)
+- assigning USB devices  (the qubes way via the tray applet)
+- audio output and input
+- PCI device 5853:0001 (Xen platform device) - no driver.
+- all other features/hardware needing special tool/driver support
+
+Installation procedure:
+- Have the Win10 ISO image (I used the 64bit version) downloaded in some qube.
+- Create a new Qube:
+  - name: Win10, colour: red
+  - standalone Qube not based on a template
+  - networking: sys-firewall (default)
+  - launch settings after creation
+  - click "OK"
+- Settings:
+  - Basic:
+    - system storage: 30000MB+
+  - Advanced:
+    - include in memory balancing: uncheck
+    - initial memory: 4096MB+
+    - Kernel: None
+    - Mode: HVM
+  - click "Apply"
+  - click "Boot from CDROM"
+    - "from file in qube":
+      - select the qube that has the ISO
+      - select ISO by clicking "..."
+    - click "OK" to boot into the windows installer
+- Windows Installer:
+  - Mostly as usual, but automatic reboots will halt the qube - just restart
+   it again and again until the installation is finished.
```

Comment:
is there a better way?

Reply:
IMO to avoid duplication with the [HVM page](https://www.qubes-os.org/doc/hvm/) it would make more sense to write that Windows 10 works as a HVM (like most other OSes) and link to the HVM page. You could then list stuff that is specific to Windows 10 (unreliable QWT so no qubes integration, ...).
My 2c :)

Reply:
my question referred to the halting, see line 52.

Reply:
Yeah I was replying to the whole PR (FWIW I wrote most of the windows doc). Overlooked the "review changes" button.

Re- your question: I don't know of any other way - once a VM shuts down (even after a reboot "event") it has to be restarted manually. Alternatively a loop and a few commands in bash can autorestart a VM after it stops running. But having to manually restarting the VM allows people to `qvm-clone` every now and then.

Reply:
Right now there is no way to avoid this manual restart. The code to automatically restart qubes-related services (gui-daemon, qrexec-daemon etc) at such restart is missing.